index.js

```diff
@@ -0,0 +1,57 @@
+const omit = require('lodash/omit');
+const { Container, config, format, transports } = require('winston');
+
+const { combine, colorize, label: addLabel, printf } = format;
+
+const container = new Container();
+const TEST = process.env.NODE_ENV === 'test';
+
+let LEVEL = 'info';
+
+const appendMeta = format(i => {
+  const meta = omit(i, ['level', 'message', 'label']);
```

Comment:
yeah but you can just write this as:

```js
const { level: _1, message: _2, label: _3, ...meta } = i;
```

it's not _that_ bad, and it's actually a ton faster than `omit()` too